build.sbt

```diff
@@ -36,7 +36,7 @@ lazy val commonSettings = Seq(
       "-Ybackend-parallelism",
       "8",
       "-Ywarn-dead-code",
-      "-Ywarn-unused:-imports,_",
```

Comment:
It turns out Scalafix simply relies on warnings about unused imports to perform the cleanup